src/Microsoft.Health.Fhir.CosmosDb/Features/Search/Queries/ExpressionQueryBuilder.cs

```diff
@@ -198,13 +198,11 @@ public object VisitBinary(BinaryExpression expression, Context context)
 
         public object VisitChained(ChainedExpression expression, Context context)
         {
-            // TODO: This will be removed once it's implemented.
             throw new SearchOperationNotSupportedException(Resources.ChainedExpressionNotSupported);
```

Comment:
>SearchOperationNotSupportedException [](start = 22, length = 36)

should this be update to `NotSupportedException`?
Same for `VisitSortParameter`

Reply:
Maybe add comment to NotSupported what chained get handled by SearchService itself and we shouldn't reach this code at all.

---
In reply to: [585065737](https://github.com/microsoft/fhir-server/pull/1695#discussion_r585065737) [](ancestors = 585065737)

Reply:
Added a comment. Keeping the existing exception for now